business/data/user/user.go

```diff
@@ -90,7 +90,7 @@ func (u User) Update(ctx context.Context, traceID string, claims auth.Claims, us
 
 	usr, err := u.QueryByID(ctx, traceID, claims, userID)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "updating user")
```

Comment:
We need to do this as well with product for the update.

Reply:
done